### ISSUE TITLE
`polygonise` doesn't work with negative values.

### ISSUE THREAD

**Issue by rafaqz** [JuliaGeo/GeometryOps.jl#140]
MWE:

```julia
using Rasters
import GeometryOps
xlon = X(525700.0:100:621100.0)
xlat = Y(5.5461e6:100:5.6031e6)
rast = fill(-1, xlon, xlat)
pol = GeometryOps.polygonize(lookup(rast, X), lookup(rast, Y), rast) # negative rastervalues return Any[]
rast = fill(10, xlon, xlat)
pol = GeometryOps.polygonize(lookup(rast, X), lookup(rast, Y), rast) #gives 1-element Vector{GeometryBasics.  ...
```

**Comment by asinghvi17** [comment]
Polygonize only works with bools as I recall

**Comment by asinghvi17** [comment]
Fixed by #141
